package.json

```diff
@@ -91,6 +92,7 @@
     "lint-staged": "^9.0.0",
     "mini-css-extract-plugin": "^0.8.0",
     "react-styleguidist": "^9.1.11",
+    "react-test-renderer": "^16.8.6",
```

Comment:
Why did you install this ?

Reply:
Doesn't look like you are using it

Reply:
Yes, when I installed `@testing-library/react-hooks` and tried to run the test, I got an error `cannot find react-test-renderer index.js  ../module...`.

I found the solution here https://github.com/testing-library/react-hooks-testing-library/issues/52